classes/form/CustomerForm.php

```diff
@@ -138,62 +138,11 @@ public function validate()
             );
             $birthdayField->setValue($dateBuilt->format('Y-m-d'));
         }
-        $this->validateFieldsLengths();
         $this->validateByModules();
 
         return parent::validate();
     }
 
-    protected function validateFieldsLengths()
```

Comment:
I don't have issue with removing the `private` methods, but these ones are `protected` so it might be considered a BC break (if the class is extended). Maybe we should just mark them as `@deprecated` ?

Reply:
Hi @jolelievre , thank you man !
Yes, this PR is a BC (see tab description) ... But except the overrides, I don't see where we can have issues 🤔 ?

Reply:
@PululuK If we can avoid BC, we avoid it. Please re-add the `protected` and trigger deprecated message. Are you ok with that @jolelievre ?

Reply:
I think just the `@deprecated` tag is enough :

```php
/**
 * @deprecated since 1.7.8.0
 */
protected function validateFieldsLengths()
{
   keep the old implementation ... etc
}
```

Because, something like that, for me it will be a BC too : 

```php
/**
 * @deprecated since 1.7.8.0
 */
protected function validateFieldsLengths()
{
    @trigger_error(sprintf('The "%s()" method is deprecated and has no effect since 1.7.7.0', __METHOD__), E_USER_DEPRECATED);
}
```
Ok for you @jolelievre @Progi1984 ?

Reply:
May be the two :


```php
/**
 * @deprecated since 1.7.8.0
 */
protected function validateFieldsLengths()
{
    @trigger_error(sprintf('The "%s()" method is deprecated and has no effect since 1.7.8.0', __METHOD__), E_USER_DEPRECATED);
   keep the old implementation ... etc
}
```

Reply:
Nice @Progi1984 !! @jolelievre  its good for you ?